awx/main/models/notifications.py

```diff
@@ -463,8 +463,8 @@ def build_notification_message(self, nt, status):
 
     def send_notification_templates(self, status):
         from awx.main.tasks import send_notifications  # avoid circular import
-        if status not in ['running', 'succeeded', 'failed']:
-            raise ValueError(_("status must be either running, succeeded or failed"))
+        if status not in ['pending', 'running', 'succeeded', 'failed']:
```

Comment:
I don't think we ended up actually reusing this method, so this change is probably not necessary.